### DISCUSSION TITLE
Chakra base component naming

### DISCUSSION THREAD

**Discussion by odinnix** [chakra-ui/chakra-ui#9094] [Ideas] [answered]
### Description

So I like the new Snippet system... but the types and naming make it almost unmaintainable. The snippet names often collide with the base-component names.

### Problem Statement/Justification

When working on a team of developers snippets are going to be very hard to maintain and force usage. Take for example Button:
- @chakra-ui/react exports a Button component
- The button snippet component is also named... Button 
- My IDE (Webstorm) will automatically import the @chakra-ui/react Button component instead of the Snippet. Additionally (and I think this is a webstorm issue) if I 'goto' the definition of Button I end up on the snippet? Which is not used. 

### Proposed Solution or API

All chakra primitive components should be namespaced or prefixed with a word so they do not get auto-improted.

**Comment by segunadebayo** [comment, marked as answer]
This will be a breaking change. I recommend using this TS plugin to sort the imports instead
https://github.com/tidalhq/ts-plugin-sort-import-suggestions

**Reply by odinnix** [reply]
I've been looking for something like this forever. Thanks!
